sdk/common/build.gradle.kts

```diff
@@ -79,4 +86,15 @@ tasks {
                 "Multi-Release" to "true"
         )
     }
-}
\ No newline at end of file
+
+    named<Test>("testResourceDisabledByProperty") {
+        jvmArgs("-Dotel.java.disabled.resource-providers=io.opentelemetry.sdk.extension.resources.OsResource,io.opentelemetry.sdk.extension.resources.ProcessResource")
+    }
+
+    named<Test>("testResourceDisabledByEnv") {
+        environment("OTEL_JAVA_DISABLED_RESOURCE_PROVIDERS", "io.opentelemetry.sdk.extension.resources.OsResource,io.opentelemetry.sdk.extension.resources.ProcessResource")
+        // Environment variable wins, this is ignored.
+        jvmArgs("-Dotel.java.disabled.resource-providers=io.opentelemetry.sdk.extension.resources.ProcessRuntimeResource")
```

Comment:
Hmm, does that mean that environment variable has precedence over system property in this case? In all other configuration props system properties > environment.

Reply:
this is a great catch, @mateuszrzeszutek ! Yes, we should be consistent and do it one way. I don't have an opinion which is better, but consistency is needed.

Reply:
Thanks a lot, must have gotten mixed up at some point! Glad I wrote these tests 😂